request_a_govuk_domain/request/templates/cookies.html

```diff
@@ -1,159 +1,85 @@
 {% extends "base.html" %}
 {% load govuk_frontend_django %}
 
-{% block title %}Cookies - Get approval to use a .gov.uk domain name - GOV.UK{% endblock %}
+{% block title %}Cookies on the Get approval to use a .gov.uk domain name service{% endblock %}
```

Comment:
The previous title was correct. Should always be in 3 parts, like all other services and pages.